libs/isograph-react-disposable-state/src/useCachedResponsivePrecommitValue.ts

```diff
@@ -51,6 +51,10 @@ export function useCachedResponsivePrecommitValue<T>(
   const lastCommittedParentCache = useRef<ParentCache<T> | null>(null);
 
   useEffect(() => {
+    if (lastCommittedParentCache.current === parentCache) {
```

Comment:
![Screenshot 2024-11-13 at 6 10 17 PM](https://github.com/user-attachments/assets/985ce3d9-c28d-48df-b113-9a920cfabb57)
I now see this, btw... I'll investigate what is going on.

My suspicion is that the component cache is caching the old component. I haven't looked into this.

I think in terms of strict mode, we're still likely incompatible. Basically, `useLazyReference` will uncommit, dispose, and re-commit, which does not re-issue the query.

I am philosophically opposed to that — queries aren't always re-issuable! But we can probably warn loudly and reissue queries. Just sucks if you're lazily doing a mutation, which is possible :-P

Once #252 lands, there probably won't be a huge need for lazy loaded queries though